src/components/InputMedia/InputMedia.module.scss

```
@@ -0,0 +1,49 @@
+@import '../../styles/global';
```

Nós temos que usar o @use ao invés do @import.

E na importação das coisas a gente usa global.$

```suggestion
@use '~styles/global';
```